### PR TITLE
DSD-1270: updated a11y docs for Notification

### PR DESCRIPTION
**Pull request by bigfishdesign13**
Fixes JIRA ticket [DSD-1270](https://jira.nypl.org/browse/DSD-1270)

## This PR does the following:

- Updates the `Accessibility` details for the `Notification` component.

## How has this been tested?

- Storybook

## Accessibility concerns or updates

- n/a

### Checklist:

- [ ] I have updated the Storybook documentation accordingly.
- [ ] I have added relevant accessibility documentation for this pull request.
- [ ] All new and existing tests passed.

### Front End Review:

- [ ] Review the Vercel preview deployment once it is ready.

### CODE REVIEW

File: src/components/Notification/Notification.stories.mdx [hunk 1]
```diff
@@ -118,16 +118,33 @@ be rendered inside other landmark elements such as the `header` and `footer`
 landmark elements. Adding a `Notification` component inside an HTML `main` landmark
 element is acceptable.
 
+**Arial Label**<br />
```
Comment (EdwinGuzman) [inline src/components/Notification/Notification.stories.mdx]: If you add a new line after the bold text, it should create the break. That's preferable over the `<br />` element.

File: src/components/Notification/Notification.stories.mdx [hunk 2]
```diff
@@ -118,16 +118,33 @@ be rendered inside other landmark elements such as the `header` and `footer`
...
 
+**Icons**<br />
 Icons rendered in the `Notification` component are decorative by default which
 means that they are hidden to screen readers. Since the "X" close icon inside the
 dismissible button is decorative and because there is no discernible text inside
 the button, an `aria-label` attribute is added to the button.
 
+**Language Patterns**<br />
+When the `Notification` component is rendered, the text within the component
+should always included wording that aligns with the type of notitication that
```
Comment (EdwinGuzman) [inline src/components/Notification/Notification.stories.mdx]: No final "d" in "included".

File: src/components/Notification/Notification.stories.mdx [hunk 3]
```diff
@@ -118,16 +118,33 @@ be rendered inside other landmark elements such as the `header` and `footer`
...
+heading or body copy within the notification should use the words `"warning"` or
+`"error"` and provide a clear description of the existing or potential isssue.
+
+**Dynamic Notifications**<br />
+If dynamically generated notifications are required, a container into which the
+notification will be injected should be included in the DOM. The container
+element should have `aria-live="polite"` applied. The container element should
+always be rendered in the DOM regardless of the presence of the Notification
+component. This DOM structure will allow the newly generated content to be
+recognized and announced by assistive technology.
```
Comment (EdwinGuzman) [inline src/components/Notification/Notification.stories.mdx:150]: Add jsx example:
```
<div aria-live="polite">
  <Notification {...props} />
</div>
```

**Review by EdwinGuzman — CHANGES_REQUESTED** [review]
Minor changes.

File: src/components/Notification/Notification.stories.mdx (hunk 1 above)
Comment (bigfishdesign13) [reply src/components/Notification/Notification.stories.mdx]: With the `<br />`, the text was not being bumped to a new line.  Two line breaks after the bold text was necessary.  But I scrapped that formatting and I am using a ### heading instead.  That said, I also added the formatting to the Resources heading.  Let's remember to do that to the same heading in other components as we move forward.

File: src/components/Notification/Notification.stories.mdx [hunk 4]
```diff
@@ -118,17 +118,44 @@ be rendered inside other landmark elements such as the `header` and `footer`
 landmark elements. Adding a `Notification` component inside an HTML `main` landmark
 element is acceptable.
 
+### Arial Label
```
Comment (EdwinGuzman) [inline src/components/Notification/Notification.stories.mdx]: Small typo: should be "ARIA".

**Review by EdwinGuzman — APPROVED** [review]

File: src/components/Notification/Notification.stories.mdx (hunk 4 above)
Comment (bigfishdesign13) [reply src/components/Notification/Notification.stories.mdx]: Ha!  We watched `The Little Mermaid` this past weekend.  It must have been on my mind.  I'll fix that.

**Pull request merged**